docs/tutorials/cli.md

```diff
@@ -148,7 +148,7 @@ This command will return the current version of Trestle and OSCAL it is using.
 
 Running `trestle version` will return:
 
-> Trestle version v2.0.0 based on OSCAL version 1.0.4
+> Trestle version v3.6.0 based on OSCAL version 1.1.2
 
```

Comment:
Should we say Trestle version v3.x.x based on OSCAL version 1.1.2
else with every release we will also have to necessarily update this.